website/versioned_docs/version-V1/network/deposits.md

```diff
@@ -175,24 +175,23 @@ This section provides a demo project that contains a detailed implementation of
 
 For running a full `omg-js` code sample for the tutorial, please use the following steps:
 
-1. Clone [OMG Samples](https://github.com/omgnetwork/omg-samples) repository:
+1. Clone [omg-js-samples](https://github.com/omgnetwork/omg-js-samples) repository:
 
 ```
-git clone https://github.com/omgnetwork/omg-samples.git
+git clone https://github.com/omgnetwork/omg-js-samples
```

Comment:
git clone does not need a .git extension ? I have not tried this myself though

Reply:
Ouch, that's me doing the mass replace. Will fix now but it will work without .git as well.